kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -72,7 +72,9 @@ def __init__(self, name, init_config, agentConfig, instances=None):
             'kube_node_status_capacity_nvidia_gpu_cards': 'node.gpu.cards_capacity',
             'kube_pod_container_status_terminated': 'container.terminated',
             'kube_pod_container_status_waiting': 'container.waiting',
+            'kube_persistentvolume_status_phase': 'persistentvolume.status',
             'kube_persistentvolumeclaim_status_phase': 'persistentvolumeclaim.status',
+            'kube_persistentvolumeclaim_resource_requests_storage_bytes': 'persistentvolumeclaim.request_storage_bytes',
```

Comment:
I'm relunctant to add yet another pvc-cardinality gauge (the existing `persistentvolumeclaim.status` is an anomaly and should have been aggregated). What is the exact use case for it?